### ISSUE TITLE
[Usable – Make Code - Simulator] Tooltip of ‘Start/stop the simulator, restart the simulator’, etc.. icon control is not visible via keyboard.

### ISSUE THREAD

**Issue by Shalini3457** [microsoft/pxt-microbit#3626]
User Experience:
User rely on keyboard will find difficulty to interact with the tooltip information if tooltip is not accessible through keyboard as user won't be able to perform functionality over such control.

Note: User credentials should NOT be included in the bug.

Repro Steps:

1. Launch application URL
2.  Home page will open.
3. Navigate "New Project" option and select it.
4. Editor screen will open.
5. Navigate to "Simulator" section appearing in left pane.
6. Navigate to "Start,Stop,Restart" option and select it.
7. Verify that user able to access all functionality.
8. Verify the tooltip of visual icon control  by keyboard.

Actual Result:
Tooltip of  icon control (Start/stop, restart, mute audio, pause execution, etc) is not visible via keyboard and due to this, end user is unable to get the context of that visual icons and also won't be able to perform functionality over such controls.

Expected Result:
Tooltip of  icon control (Start/stop, restart, mute audio, pause execution, etc)  should be displayed via keyboard, so that User rely on keyboard will not find difficulty to interact with the tooltip information and perform functionality over such control.

Note:
This issue also exists with Firefox and Chrome Browser.

Test Environments:
Version: 20H2 (OS Build: 19042.508)
Browser: Edge chromium DEV Build (Version 87.0.654.0), Firefox(75.0), Chrome

**Comment by Shalini3457** [comment]
Labels:
#Accessibility, #A11ySev4, #HCL, #Benchmark-HCL-MakeCode-OCT2020, #Win10-Edge(Chromium)

[Usable_Tooltip is not visible in keyboard navigation.zip](https://github.com/microsoft/pxt-microbit/files/5509614/Usable_Tooltip.is.not.visible.in.keyboard.navigation.zip)

**Comment by jwunderl** [comment]
same as https://github.com/microsoft/pxt-microbit/issues/3582, fix appears to be coming on browser side.
